sinks/api/v1/supported_metrics.go

```diff
@@ -66,8 +66,7 @@ var statMetrics = []SupportedStatMetric{
 			return spec.HasCpu && (spec.Cpu.Limit > 0)
 		},
 		GetValue: func(spec *cadvisor.ContainerSpec, stat *cadvisor.ContainerStats) []InternalPoint {
-			// Normalize to a conversion factor of 1000.
-			return []InternalPoint{{Value: int64(spec.Cpu.Limit*1000) / 1024}}
+			return []InternalPoint{{Value: int64(spec.Cpu.Limit)}}
```

Comment:
I would have expected we still do this? We fixed it for node containers but not for general containers no?